### PR TITLE
Feature/hit the api

### PR DESCRIPTION
**Pull request by tnyandoro**
Project requirements
Read the Leaderboard API documentation to learn how to use this API.
Create a new game with the name of your choice by using the API.
Implement the "Refresh" button (receiving data from the API and parsing the JSON).
Implement the form "Submit" button (sending data to the API).
Use async and await JavaScript features to consume the API.
No styling is required.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,9 +4,10 @@
   "description": "IMPORTANT NOTE This is the first time you will use Gitflow. Keep that in mind! Create an npm project with webpack. (If you need a refresher, check the previous project or webpack exercise). Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps. Use this wireframe as a guide:.Don't make API calls (you'll do it in a following lesson). Use ES6 modules, with import and export.",
   "main": "index.js",
   "scripts": {
-    "start": "webpack serve",
+    "start": "webpack",
```
Comment (vichuge) [inline package.json]: - [ ] I just clone the repo, then I use `npm i` to install everything and then I tried `npm start`, unfortunately, the app doesn't start, I tried reading the readme file if there's a different to proceed but it said using `npm start` is the way to do it. I kindly invite you to change your `package.json` way to start the project. 🐱 
- Hint: You could try `"start": "webpack serve --open",`
image

File: src/apiData.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  const result = await fetch(`${API_URL}/games/${GAME_ID}/scores`, {
+    method: 'POST',
+    body: JSON.stringify(data),
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+  }).then((response) => response.json());
+  return result;
+};
```
Comment (vichuge) [inline src/apiData.js:19]: - I like this way to use the promises, excellent job with this! 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tnyandoro** [comment]
##  Changes Added
--npm run start
Project is running at http://localhost:8080/
or 
--npm run server
in your browser go to http://localhost:3000
to run the project

**Review by bellom — APPROVED** [review]
## Hi @tnyandoro  👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**